### DISCUSSION TITLE
Re-sizable debug window

### DISCUSSION THREAD

**Discussion by AstralButOnGitHub** [ScratchAddons/ScratchAddons#5221] [Ideas]
What if there was a Re sizable debug window?
I think it would look like this:
![Hello!](https://user-images.githubusercontent.com/116039257/196294276-085154b2-d2c2-46fd-abbb-8d8227b3b35a.png)

**Comment by BroJac5246** [comment]
I don't see why not. You're welcome to create an issue suggesting it, and, if you know how and want to, create a PR.

**Reply by AstralButOnGitHub** [reply]
I don't know how to code XD
(Except in Scratch)

**Reply by BroJac5246** [reply]
You're still welcome to create an issue, someone will probably add it at some point :)

**Reply by AstralButOnGitHub** [reply]
Ok :)
